test/Test/Reference.hs

```diff
@@ -131,7 +116,7 @@ randomParams = do
 
 rndCircuit :: Gen (ArithCircuit Fr, Assignment Fr)
 rndCircuit = do
-  n <- choose (1, 20)
+  n <- choose (1, 1)
```

Comment:
This seems small? It always returns 1?

Reply:
Fixed. Thank you. I was testing on small values of n and forgot to change it back